pootle/checks.py

```diff
@@ -262,6 +262,50 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W015",
                 ))
 
+    if settings.POOTLE_TM_SERVER:
+        tm_indexes = []
+
+        if 'local' not in settings.POOTLE_TM_SERVER:
+            errors.append(checks.Critical(
+                _("POOTLE_TM_SERVER['local'] is missing.", server),
```

Comment:
Seriously I don't think we even need to warn.  There might be a reason you don't have it on.  We should simply have a good basic config, we do.  Work correctly without any present, I think we do.